.github/workflows/handler-test.yml

```
@@ -60,6 +78,36 @@ jobs:
           echo "Curling \`health_check_url\` for a return status of 200..."
           while ! curl -sfS --max-time 5 $( terraform output health_check_url ); do sleep 5; done
 
+      - name: Retrieve TFE URL
+        id: retrieve-tfe-url
+        run: |
+          terraform output -no-color -raw tfe_url
+
+      - name: Retrieve IACT Token
+        id: retrieve-iact-token
+        run: |
+          curl $( terraform output -no-color -raw iact_url )
+
+      - name: Create Admin in TFE
+        id: create-admin
+        run: |
+          curl \
+            --header "Content-Type: application/json" \
+            --data '{ "username": "tester", "email": "tf-onprem-team@hashicorp", "password": "${{ secrets.TFE_PASSWORD }}" }' \
+            $( terraform output -no-color -raw initial_admin_user_url )?token=${{ steps.retrieve-iact-token.stdout }} \
+            | jq --raw-output '.token'
+
+      - name: Run k6 Smoke Tests
+        id: run-smoke-tests
+        working-directory: *K6_WORK_DIR
+        env:
+          K6_PATHNAME: "./k6"
+          TFE_URL: "${{ steps.retrieve-tfe-url.stdout }}"
+          TFE_API_TOKEN: "${{ steps.retrieve-iact-token.stdout }}"
```

This should refer to the token created along for the new admin user:

```suggestion
          TFE_API_TOKEN: "${{ steps.create-admin.stdout }}"
```